presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -225,6 +228,20 @@ public SystemSessionProperties(
                         COLOCATED_JOIN,
                         "Experimental: Use a colocated join when possible",
                         featuresConfig.isColocatedJoinsEnabled(),
+                        false),
+                new PropertyMetadata<>(
+                        OPERATOR_MEMORY_LIMIT_BEFORE_SPILL,
+                        "Experimental: Operator memory limit before spill",
+                        VARCHAR,
+                        DataSize.class,
+                        featuresConfig.getOperatorMemoryLimitBeforeSpill(),
+                        false,
+                        value -> DataSize.valueOf((String) value),
+                        DataSize::toString),
+                longSessionProperty(
+                        MAX_ENTRIES_BEFORE_SPILL,
+                        "Experimental: Limit max number of entries before spill",
```

Comment:
We do a lot of tests by setting memory limits to 1byte, and for cases that are a little more complex you can usually estimate the amount required. For example, with 100 groups and a COUNT aggregation, you'll need something like 2-3KB of memory (8 bytes for COUNT state, and the rest for hash table)